docs/guide/types/ellipse.md

```diff
@@ -92,4 +97,10 @@ If one of the axes does not match an axis in the chart, the ellipse will take th
 | `borderWidth` | Stroke width
 | `borderDash` | Length and spacing of dashes. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/setLineDash).
 | `borderDashOffset` | Offset for line dashes. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineDashOffset).
+| `borderShadowColor` | The color of the border shadow. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/shadowColor).
 | `backgroundColor` | Fill color
+| `backgroundShadowColor` | The color of shadow. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/shadowColor).
```

Comment:
same here, background should be before border

Reply:
All the options in the doc must be reordered because they are not sorted alphabetically. It was already in my TODO list.

Reply:
a specific PR is needed